### PR TITLE
Add /about with final content

### PR DESCRIPTION
**Pull request by makaelastephens**
## Ticket

https://wicmtdp.atlassian.net/browse/PRP-217

## Changes
Added `/about` to explain the recertification process.

## Context for reviewers
Rocket suggested that rather than try to delete `/intro` to just add a new page. Deleting `/intro` can be handled separately.

## Testing
image

### CODE REVIEW

**Comment by makaelastephens** [comment]
@microwavenby This PR passed the playwright checks, is it because no snapshots exist since this is a new page?

**Review by microwavenby — APPROVED** [review]
My only comment is that I'd like personally prefer the sections to be nested in the localization file, but this is a nit

```json
 {
   "About":{
      "title":"How it works",
      "note":"Note: This form does not guarantee your enrollment in WIC. Your eligibility will be confirmed when you have your recertification appointment with WIC.",
      "button":"Continue",
      "answer":{
         "body":"We only need to collect information about WIC participants who have a recertification appointment.",
         "header":"Answer questions about who is recertifying in your household"
      },
      "upload":{
         "body":"We will help you figure out which ones you need. If you usually have to show income information, like paystubs or tax returns, have those documents accessible before you start.",
         "header":"Upload any needed documents"
      },
      "appointment":{
         "body":"Your local WIC clinic staff will confirm the information and documents you sent.",
         "header":"Later, attend your recertification appointment"
      }
   }
}
```

File: participant/app/routes/about.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { Button } from "@trussworks/react-uswds";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+
+export default function Index() {
+  const { t } = useTranslation();
+  const listProcessKeys: string[] = ["answer", "upload", "appointment"];
+
+  return (
+    <div className="measure-6">
```
Comment (rocketnova) [inline participant/app/routes/about.tsx]: @makaelastephens Is this the measure we want on all the pages? If so, we might want to put it in the Layout

**Review by rocketnova — COMMENTED** [review]
@makaelastephens 

> This PR passed the playwright checks, is it because no snapshots exist since this is a new page?

This PR passed the playwright checks because:

1. This is a new page and it has no snapshots to test against
2. This new page has no playwright tests

I overlooked adding an e2e test for to this ticket's acceptance criteria. Would you mind adding one in? It should look something like `participant/e2e/routes/index.spec.ts` and be named `participant/e2e/routes/about.spec.ts`

File: participant/app/routes/about.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          <li className="usa-process-list__item" key={key}>
+            <h2 className="usa-process-list__heading">
+              <Trans i18nKey={`About.${key}Header`} />
+            </h2>
+            <p className="margin-top-1">
+              <Trans i18nKey={`About.${key}`} />
+            </p>
+          </li>
+        ))}
+      </ol>
```
Comment (microwavenby) [inline participant/app/routes/about.tsx]: We should be using Truss components here, not using the classes to imitate the components..

https://trussworks.github.io/react-uswds/?path=/docs/components-processlist--process-list-default
There's a `<ProcessList>` component, so we could do

```suggestion
      <ProcessList>
        {listProcessKeys.map((key: string) => (
          <ProcessListItem key={key}>
            <ProcessListHeading type="h2">
              <Trans i18nKey={`About.${key}Header`} />
            </ProcessListHeading>
            <p className="margin-top-1">
              <Trans i18nKey={`About.${key}`} />
            </p>
          </ProcessListItem>
        ))}
      </ProcessList>
```

File: participant/app/routes/about.tsx (hunk 1 above)
Comment (makaelastephens) [reply participant/app/routes/about.tsx]: Yeah I think we should use 6 as the base. @microwavenby do you want me to make a ticket for updating the layouts? Or is it simple to add that to this PR?

File: participant/app/routes/about.tsx (hunk 2 above)
Comment (makaelastephens) [reply participant/app/routes/about.tsx]: Ah thanks, I was looking at the eligibility screener as an example

**Pull request merged**